### DISCUSSION TITLE
Use stimulus from child view (partials)

### DISCUSSION THREAD

**Discussion by renzodiaz** [excid3/tailwindcss-stimulus-components#54] [General]
Just want to ask if there is a way to execute an action from <%= render partial: 'child'  %>, for example when you are building a sidebar that hides and shows you usually use partials instead putting all the code in the layout.

**Comment by excid3** [comment]
Not entirely following your question, but you can use Stimulus controllers anywhere as they're just applied to your HTML. Post some more details if you have an example you're trying to figure out how to implement. 👍

**Reply by renzodiaz** [reply]
I'm using Jumpstart Pro with tailwind, what I'm using a dashboard layout from tailwind components, what happens is that I have my sidebar menu in a partial component but in mobile the button to open the sidebar menu is under other partial called page_header when I click on that button it throws an error in the console `Error: Missing target element "slideover.overlay"`. [Layout](https://gist.github.com/renzodiaz/922bcb56d121ec360ea9255fccaf3788)

**Reply by excid3** [reply]
Stimulus will require all the targets be in the HTML, so you can't dynamically load those.

You didn't include any code that uses Stimulus, so not a lot I can help with there.

**Reply by renzodiaz** [reply]
Yeah, that's what happens all my targets and action should be in the same context, I'll change my layout. Thanks Chris.

**Reply by excid3** [reply]
Cool, let me know if that works. 👍
